README.md

```diff
@@ -43,10 +45,33 @@ Exec type options:<br />
 The "Settings" window on a mod displays the Command Line to play that mod directly via Quick Launch. You can copy that Quick Launch Command Line and use it to create a shortcut to easily launch that mod. <br />
 
 <br /><br />
-## **Software Rendering Mode:**<br />
-By default KnossosNET will render the UI in the GPU, can be set to run completely on software rendering what effectively avoids any use of the user GPU, this will come at the cost of increased CPU usage, what should not be a problem when in idle.
+## **Tool Quick Launch:**<br />
+Just like mods QuickLaunch, you can launch installed mod development tools with a cmdline argument. Windows users have the possibility to create a shortcut to desktop with a click. This may be extended to Linux and Mac later on.<br />
+KnossosNET.exe -tool {tool-id}
+
+<br /><br />
+## **Tray Icon Mode:**<br />
+On 1.3.0 a feature to "close to tray" has been implemented, this can be enabled on global settings.<br />
+Additionally, a cmdline argument (-traymode) has been added to Knossos.NET. By starting in this mode the launcher will start to tray and will not load UI resources until the user opens the UI for the first time.<br />
+The tray icon allow to launch mods directly from there.<br />
+![TrayIcon](https://i.imgur.com/cWF4WdT.png)
+
+<br /><br />
+## **Rendering Mode:**<br />
+From 1.3.0 KnossosNET will render the UI in software mode by default to avoid use of the user GPU, but it can be set to run on hardware gpu rendering.
```

Comment:
Reccomend capitalizating the second use of the word `gpu` to match the first usage. 

IE `hardware gpu rendering.` should ideally be `hardware GPU rendering.`